src/databricks/labs/ucx/hive_metastore/workflows.py

```diff
@@ -12,29 +12,48 @@ def __init__(self):
     def migrate_external_tables_sync(self, ctx: RuntimeContext):
         """This workflow task migrates the *external tables that are supported by SYNC command* from the Hive Metastore to the Unity Catalog.
         Following cli commands are required to be run before running this task:
-        - For Azure: `principal-prefix-access`, `create-table-mapping`, `create-uber-principal`, `migrate-credentials`, `migrate-locations`, `create-catalogs-schemas`
-        - For AWS: TBD
+        - For Azure or AWS: `principal-prefix-access`, `create-table-mapping`, `create-uber-principal`, `migrate-credentials`, `migrate-locations`, `create-catalogs-schemas`
         """
-        ctx.tables_migrator.migrate_tables(what=What.EXTERNAL_SYNC, acl_strategy=[AclMigrationWhat.LEGACY_TACL])
+        ctx.tables_migrator.migrate_tables(
+            what=What.EXTERNAL_SYNC, acl_strategy=[AclMigrationWhat.LEGACY_TACL, AclMigrationWhat.PRINCIPAL]
+        )
 
     @job_task(job_cluster="table_migration", depends_on=[Assessment.crawl_tables])
     def migrate_dbfs_root_delta_tables(self, ctx: RuntimeContext):
         """This workflow task migrates `delta tables stored in DBFS root` from the Hive Metastore to the Unity Catalog using deep clone.
         Following cli commands are required to be run before running this task:
-        - For Azure: `principal-prefix-access`, `create-table-mapping`, `create-uber-principal`, `migrate-credentials`, `migrate-locations`, `create-catalogs-schemas`
-        - For AWS: TBD
+        - For Azure or AWS: `principal-prefix-access`, `create-table-mapping`, `create-uber-principal`, `migrate-credentials`, `migrate-locations`, `create-catalogs-schemas`
+        """
+        ctx.tables_migrator.migrate_tables(
+            what=What.DBFS_ROOT_DELTA, acl_strategy=[AclMigrationWhat.LEGACY_TACL, AclMigrationWhat.PRINCIPAL]
+        )
+
+    @job_task(job_cluster="table_migration", depends_on=[Assessment.crawl_tables])
+    def migrate_dbfs_root_non_delta_tables(self, ctx: RuntimeContext):
+        """This workflow task migrates `non delta tables stored in DBFS root` from the Hive Metastore to the Unity Catalog using CTAS.
```

Comment:
could you standardise the formatting of these docstrings? we currently have 3 different styles
- This workflow task migrates the *external tables that are supported by SYNC command* 
- This workflow task migrates `delta tables stored in DBFS root`
- This workflow task migrates views from the Hive Metastore

Reply:
updated formatting. Also removed the requirement for pre-requisite cli cmd, it was repetitive and also mentioned in the read me doc